### PR TITLE
Restructuring project structure

### PR DESCRIPTION
**Pull request by SanchayGupta1197**
#### Pull Request Description

Restructuring the project structure in order to align it with confluent connector practices

---

**Breaking Changes:**
- changing the way the project is built in line with confluent connectors
- removed org.testing dependencies and remapped them to suitable junit methods

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,34 +8,74 @@
...
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <version>0.11.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <title>Kafka Connect Azure Kusto Connector</title>
+                            <version>${project.version}-preview</version>
```
Comment (fahadhasher) [inline pom.xml:50]:
```suggestion
                            <version>${project.version}</version>
```

File: pom.xml [hunk 2]
```diff
@@ -8,34 +8,74 @@
...
+                            <ownerUsername>Microsoft</ownerUsername>
```
Comment (fahadhasher) [inline pom.xml:51]:
```suggestion
                            <ownerUsername>Microsoft Corporation</ownerUsername>
```

File: pom.xml [hunk 3]
```diff
@@ -8,34 +8,74 @@
...
+                            <ownerName>Microsoft.</ownerName>
```
Comment (fahadhasher) [inline pom.xml:52]:
```suggestion
                            <ownerName>Microsoft Corporation</ownerName>
```

File: pom.xml [hunk 4]
```diff
@@ -8,34 +8,74 @@
...
+                            <componentTypes>
+                                <componentType>sink</componentType>
+                            </componentTypes>
+                            <tags>
+                                <tag>Azure Kusto </tag>
```
Comment (fahadhasher) [inline pom.xml:57]:
```suggestion
                                <tag>Azure</tag>
                                <tag>Kusto</tag>
                                <tag>Data Explorer</tag>
                                <tag>Database</tag>
```

Comment (SanchayGupta1197) [reply pom.xml:57]: done

File: pom.xml (hunk 3 above)
Comment (SanchayGupta1197) [reply pom.xml:52]: done

File: pom.xml (hunk 2 above)
Comment (SanchayGupta1197) [reply pom.xml:51]: done

File: pom.xml (hunk 1 above)
Comment (SanchayGupta1197) [reply pom.xml:50]: done

**Pull request merged**